tools/sdk-ai-bots/tests.yml

```diff
@@ -0,0 +1,117 @@
+parameters:
+- name: PythonVersion
+  type: string
+  default: '3.10'
+
+trigger:
+  branches:
+    include:
+      - azure-sdk-ai-bot
+      - main
+  paths:
+    include:
+      - tools/sdk-ai-bots/azure-sdk-qa-bot-backend/
+
+pr:
+  branches:
+    include:
+      - azure-sdk-ai-bot
+      - azure-sdk-ai-bot-dev
+      - main
+  paths:
+    include:
+      - tools/sdk-ai-bots/azure-sdk-qa-bot-backend/
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    Use1ESOfficial: false
+    stages:
+      - stage: 'evaluation'
+        variables:
+          - template: /eng/pipelines/templates/variables/globals.yml
+          - template: /eng/pipelines/templates/variables/image.yml
+        pool:
+              name: $(LINUXPOOL)
+              image: $(LINUXVMIMAGE)
+              os: linux
+        jobs:
+          - job: 'evaluation'
+            displayName: 'evaluation'
+            steps:
+              - template: /eng/pipelines/templates/steps/use-python-version.yml
+                parameters:
+                  versionSpec: '${{ parameters.PythonVersion }}'
+              - script: |
+                  python --version
+                displayName: 'check python'
+              - script: |
+                  echo "Downloading Go 1.23.0..."
+                  curl -LO https://go.dev/dl/go1.23.0.linux-amd64.tar.gz
+
+                  echo "Removing existing Go installation..."
+                  sudo rm -rf /usr/local/go
+
+                  echo "Extracting Go 1.23.0..."
+                  sudo tar -C /usr/local -xzf go1.23.0.linux-amd64.tar.gz
+
+                  echo "Setting environment variables..."
+                  echo "##vso[task.setvariable variable=GOROOT]/usr/local/go"
+                  echo "##vso[task.prependpath]/usr/local/go/bin"
+                displayName: 'install go'
+              - script: |
+                  go version
+                displayName: 'check go'
+              - task: AzureCLI@2
+                inputs:
+                  azureSubscription: 'azure-chat-bot-test'
+                  addSpnToEnvironment: true
+                  scriptType: bash
+                  scriptLocation: inlineScript
+                  inlineScript: |
+                    echo "##vso[task.setvariable variable=AZURE_CLIENT_ID]$servicePrincipalId"
+                    echo "##vso[task.setvariable variable=AZURE_TENANT_ID]$tenantId"
+                    echo "##vso[task.setvariable variable=AZURE_SUBSCRIPTION_ID]$(az account show --query id -o tsv)"
+                    echo "$idToken" > "$(Agent.BuildDirectory)/id_token"
+                    echo "##vso[task.setvariable variable=AZURE_FEDERATED_TOKEN_FILE]$(Agent.BuildDirectory)/id_token"
+                    echo "##vso[task.setvariable variable=SYSTEM_ACCESSTOKEN]$idToken"
```

Comment:
System.AccessToken is a predefined DevOps variable I suspect this will not actually set correctly here.